odoo/tests/common.py

```diff
@@ -2490,12 +2527,16 @@ def _is_matching(test_filter):
                 return False
             elif method and test_method and method != test_method:
                 return False
+            if parameter:
+                test._test_params.append(parameter)
             return True
 
-        if any(_is_matching(test_filter) for test_filter in self.exclude):
+        if any(_match(test_filter) for test_filter in self.exclude):
             return False
```

Comment:
That seems like a potential problem (or at least an odd variability) as in case of exclusion it's only going to partially collect parameters.

Reply:
Check this part: https://github.com/odoo/odoo/pull/96228/files#diff-80aef099e5fb41cc09e699d35b749152aca50caf5b0bc16c400081c29023fbecR2493-R2494

If this is an exclusion, the test won't be ran, and the parameters are useless.